admin/app/controllers/spree/admin/return_index_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Spree
```

Comment:
@Vegann we don't need this controller, please split this into

`ReturnsAuthorizationsController#index`
and `CustomerReturns#index` to be pure RESTful